components/renku_data_services/users/api.spec.yaml

```diff
@@ -417,28 +417,34 @@ components:
         id:
           $ref: "#/components/schemas/Ulid"
         name:
-          $ref: "#/components/schemas/SecretName"
+         $ref: "#/components/schemas/SecretName"
+        default_filename:
+          $ref: "#/components/schemas/SecretDefaultFilename"
```

Comment:
question: Why not make this part of the session secret slot? This can be paired to a data connector and then the file name does not make a lot of sense. In the ORM object for a secret we have a key for each secret, can we not default to having the key be the default filename if we need a file?

Reply:
This is for Renku 1.0 sessions as explained in the PR description. The field is ignored for Renku 2.0.